kapitel/kapitel1.tex

```diff
@@ -0,0 +1,54 @@
+Außerdem ist es vorgesehen, dass Makler Angebote kopieren können. Dies beinhaltet die gleiche Logik wie die Angebotsspeicherung, bis auf dass eine neue \ac{GUID} erzeugt wird, wodurch dementsprechend ein neuer Eintrag in der Datenbank und im Maklerportal hinterlegt wird. Außerdem muss mit der \ac{GUID}, bzw. Session der Kopie weitergearbeitet werden.\\
+Die \ac{KM} bietet auch \ac{OTR} außerhalb des Maklerportals an. Für diese Rechner ist dann keine Angebots-, oder Kopiespeicherung vorgesehen, dementsprechend sind die Knöpfe dort nicht vorhanden. \\
+
+% Drucken
+Die Druckenfunktionalität wird durch zwei Knöpfe abgebildet, einen um ein Angebot zu drucken, der zweite für den Antrag. Ein Angebot wird z.B. dem Endkunden vom Makler unterbreitet und ein Antrag ist ein verbindliches Dokument mit welchem ein Versicherungsvertrag abgeschlossen werden kann. Somit ist diese Funktionalität sehr wichtig, als auch die Korrektheit der Daten in diesen Dokumenten. Ein Dokument soll dann gedrückt werden können, wenn Beiträge erfolgreich berechnet wurden. Dafür müssen alle berechnungsrelevanten Angaben getroffen worden sein. Die Knöpfe sind bis das geschehen ist deaktiviert und werden reaktiv aktiviert. Außerdem kann es sein, dass für einen Tarif der Angebot- oder Antragsdruck deaktiviert ist, wo dann natürlich die Knöpfe auch dementsprechend aktiv sind. Das Drucken ist dabei eigentlich das Herunterladen eines PDF-Dokumentes, damit der Makler dieses auch per E-Mail verschicken oder selber ausdrucken kann. Das Dokument wird von einem angebundenem System anhand der Daten in XML-Format generiert.
+
+%Versand
+Eine weitere Funktion ist die des Dokumentenversands. Bei Drücken dieses Knopfs öffnet sich ein Modal wo der Nutzer die gewünschten Dokumente aus allen relevanten Dokumenten auswählen kann. Dazu muss noch eine, oder mehrere E-Mails angegeben werden und dann werden die Dokumente per E-Mail in einem ZIP-Archiv versendet. Die Dokumente beinhalten rechtliche Informationen und der Antrag, sowie das Angebot können auch ausgewählt werden, mit der gleichen Einschränkung wie bei der Druckfunktionalität beschrieben. Die komplette Dokumentenversandkomponente gibt es als eigenständige Bibliothek, da sie auch für andere \ac{OTR} genutzt wird, so dass diese für Frontend nur eingebunden werden musste.
+
+\subsection{Projektbegründung}
+\label{projektbegründung}
+Die Motivation hinter diesem Projekt begründet sich in der einfachen Erweiterbarkeit für neue Versicherungsprodukte, so wie eine Überholung des Aussehens des \ac{OTR}. Allgemein ist der \ac{OTR} durch den modularen und komponentenbasierten Aufbau des Projekts besser wartbar und Einzelteile einfacher auszutauschen. \\
+Die Fußzeile ist eine Komponente die in allen bestehenden Produkten und zukünftig kommenden Produkten vorhanden sein muss. Die Position bietet sich außerdem für die gegeben Funktionalitäten gut an, da sie unabhängig von dem Kontext der Rest der Webseite funktionieren soll.
```

Comment:
"Die Position bietet sich außerdem für die gegeben Funktionalitäten gut an, ..." würde ich umschreiben zu "Die Platzierung der gegebenen Funktionalitäten in die Fußzeile bietet sich dafür gut an,"